<!DOCTYPE html>
<html >
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
      <title>Publications</title>
    
      <link rel="stylesheet" href="_static/pygments.css">
      <link rel="stylesheet" href="_static/theme.css">
      <link rel="stylesheet" href="_static/sphinx_press_theme.css">
      
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>

      <!-- sphinx script_files -->
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
        <script src="_static/language_data.js"></script>

      
      <script src="_static/theme-vendors.js"></script>
      <script src="_static/theme.js" defer></script>
    
  <link rel="author" title="About these documents" href="about.html" />
  <link rel="index" title="Index" href="genindex.html" />
  <link rel="search" title="Search" href="search.html" />
  <link rel="next" title="Members" href="community/members.html" />
  <link rel="prev" title="Spatiocyte" href="projects/spatiocyte.html" /> 
  </head>

  <body>
    <div id="app" class="theme-container" :class="pageClasses"><navbar @toggle-sidebar="toggleSidebar">
  <router-link to="index.html" class="home-link">
    
      <span class="site-name">E-Cell Project</span>
    
  </router-link>

  <div class="links">
    <navlinks class="can-hide">

  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link ">
         about
      </a>
    </div>
  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link ">
         projects
      </a>
    </div>
  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link  router-link-active">
         publications
      </a>
    </div>
  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link ">
         community
      </a>
    </div>
  



    </navlinks>
  </div>
</navbar>

      
      <div class="sidebar-mask" @click="toggleSidebar(false)">
      </div>
        <sidebar @toggle-sidebar="toggleSidebar">
          
          <navlinks>
            

  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link ">
         about
      </a>
    </div>
  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link ">
         projects
      </a>
    </div>
  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link  router-link-active">
         publications
      </a>
    </div>
  
    <div class="nav-item">
      <a href="index.html#what-is-e-cell-project"
         class="nav-link ">
         community
      </a>
    </div>
  



            
          </navlinks><div class="sidebar-links" role="navigation" aria-label="main navigation">
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="index.html#what-is-e-cell-project">about</a></span>
      </p>
      <ul class="">
        
          <li class="toctree-l1 "><a href="about.html" class="reference internal ">What is E-Cell System?</a>

            
          </li>

        
      </ul>
    </div>
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="index.html#what-is-e-cell-project">projects</a></span>
      </p>
      <ul class="">
        
          <li class="toctree-l1 "><a href="projects/ecell4.html" class="reference internal ">The E-Cell System version 4</a>

            
          </li>

        
          <li class="toctree-l1 "><a href="projects/ecell3.html" class="reference internal ">The E-Cell System version 3</a>

            
          </li>

        
          <li class="toctree-l1 "><a href="projects/scopyon.html" class="reference internal ">Scopyon</a>

            
          </li>

        
          <li class="toctree-l1 "><a href="projects/spatiocyte.html" class="reference internal ">Spatiocyte</a>

            
          </li>

        
      </ul>
    </div>
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="index.html#what-is-e-cell-project">publications</a></span>
      </p>
      <ul class="current">
        
          <li class="toctree-l1 current"><a href="#" class="reference internal current">Publications</a>

            
              <ul>
                
                  <li class="toctree-l2"><a href="#publications" class="reference internal">Publications</a></li>
                
              </ul>
            
          </li>

        
      </ul>
    </div>
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="index.html#what-is-e-cell-project">community</a></span>
      </p>
      <ul class="">
        
          <li class="toctree-l1 "><a href="community/members.html" class="reference internal ">Members</a>

            
          </li>

        
          <li class="toctree-l1 "><a href="community/community.html" class="reference internal ">Developer team</a>

            
          </li>

        
      </ul>
    </div>
  
</div>
        </sidebar>

      <page>
          <div class="body-header" role="navigation" aria-label="navigation">
  
  <ul class="breadcrumbs">
    <li><a href="index.html">Docs</a> &raquo;</li>
    
    <li>Publications</li>
  </ul>
  

  <ul class="page-nav">
  <li class="prev">
    <a href="projects/spatiocyte.html"
       title="previous chapter">← Spatiocyte</a>
  </li>
  <li class="next">
    <a href="community/members.html"
       title="next chapter">Members →</a>
  </li>
</ul>
  
</div>
<hr>
          <div class="content" role="main">
            
  <div class="section" id="publications">
<h1>Publications<a class="headerlink" href="#publications" title="Permalink to this headline">¶</a></h1>
<ul>
<li><p>Shimo H, Arjunan SNV, Machiyama H, Nishino T, Suematsu M, Fujita H,
Tomita M, Takahashi K: Particle Simulation of Oxidation Induced Band
3 Clustering in Human Erythrocytes. <em>PLoS computational
biology</em>. 11:e1004210
(2015) <a class="reference external" href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004210">Open_Access</a></p></li>
<li><p>Nishino T, Yachie-Kinoshita A, Hirayama A, Soga T, Suematsu M,
Tomita M. Dynamic Simulation and Metabolome Analysis of Long-Term
Erythrocyte Storage in Adenine–Guanosine Solution. <em>PLoS One</em>.
<strong>8</strong>(8):e71060 (2013)
<a class="reference external" href="http://dx.doi.org/10.1371/journal.pone.0071060">Open_Access</a></p></li>
<li><p>Okubo C, Sano HI, Naito Y, Tomita M. Contribution of quantitative
changes in individual ionic current systems to the embryonic
development of ventricular myocytes: a simulation study. <em>J Physiol
Sci</em>. <strong>63</strong>(5):355-67 (2013)
<a class="reference external" href="http://dx.doi.org/10.1007/s12576-013-0271-x">Open_Access</a>
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/23760774">PubMed</a></p></li>
<li><p>Mugler A, Bailey AG, Takahashi K, ten Wolde PR. Membrane clustering
and the role of rebinding in biochemical signaling. <em>Biophys J</em>.
<strong>102</strong>(5):1069-78 (2012)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/22404929">PubMed</a></p></li>
<li><p>Seike M, Saitou T, Kouchi Y, Ohara T, Matsuhisa M, Sakaguchi K,
Tomita K, Kosugi K, Kashiwagi A, Kasuga M, Tomita M, Naito Y,
Nakajima H. Computational assessment of insulin secretion and
insulin sensitivity from 2-h oral glucose tolerance tests for
clinical use for type 2 diabetes. <em>J Physiol Sci</em>
<strong>61</strong>(4):321-30 (2011)
<a class="reference external" href="http://dx.doi.org/10.1007/s12576-011-0153-z">Open_Access</a>
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/21594678">PubMed</a></p></li>
<li><p>Shimo H, Nishino T, Tomita M. Predicting the Kinetic Properties
Associated with Redox Imbalance after Oxidative Crisis in
G6PD-Deficient Erythrocytes: A Simulation Study. <em>Adv Hematol</em>
<strong>2011</strong>:398945 (2011)
<a class="reference external" href="http://dx.doi.org/10.1155/2011/398945">Open_Access</a>
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/21977034">PubMed</a></p></li>
<li><p>Takahashi K, Tanase-Nicola S, ten Wolde PR. Spatio-temporal
correlations can drastically change the response of a MAPK pathway.
<em>Proc Natl Acad Sci U S A</em> <strong>107</strong>(6):2473-8 (2010)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/20133748">PubMed</a></p></li>
<li><p>Yachie-Kinoshita A, Nishino T, Shimo H, Suematsu M, Tomita M. A
metabolic model of human erythrocytes: practical application of the
E-Cell Simulation Environment. <em>J Biomed Biotechnol</em>
<strong>2010</strong>:642420 (2010)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/20625505">PubMed</a></p></li>
<li><p>Nishino T, Yachie-Kinoshita A, Hirayama A, Soga T, Suematsu M,
Tomita M. In silico modeling and metabolome analysis of long-stored
erythrocytes to improve blood storage methods. <em>J Biotechnol</em>
<strong>144</strong>(3):212-23 (2009)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/19695295">PubMed</a></p></li>
<li><p>Helmy M, Gohda J, Inoue J, Tomita M, Tsuchiya M, Selvarajoo K.
Predicting novel features of toll-like receptor 3 signaling
in macrophages. <em>PLoS One</em> <strong>4</strong>(3):e4661 (2009)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/19252739">PubMed</a></p></li>
<li><p>Ogawa Y, Arakawa K, Kaizu K, Miyoshi F, Nakayama Y, Tomita M.
Comparative study of circadian oscillatory network models
of Drosophila. <em>Artif Life</em> <strong>14</strong>(1):29-48 (2008)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/18171129">PubMed</a></p></li>
<li><p>Ohno H, Naito Y, Nakajima H, Tomita M. Construction of a biological
tissue model based on a single-cell model: a computer simulation of
metabolic heterogeneity in the liver lobule. <em>Artif Life</em>
<strong>14</strong>(1):3-28 (2008)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/18171128">PubMed</a></p></li>
<li><p>Selvarajoo K, Takada Y, Gohda J, Helmy M, Akira S, Tomita M,
Tsuchiya M, Inoue J, Matsuo K. Signaling flux redistribution at
toll-like receptor pathway junctions. <em>PLoS One</em>
<strong>3</strong>(10):e3430 (2008)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/18927610">PubMed</a></p></li>
<li><p>Ishii N, Nakayama Y, Tomita M. Distinguishing enzymes using
metabolome data for the hybrid dynamic/static method. <em>Theor Biol
Med Model</em> <strong>4</strong>:19 (2007)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/17511884">PubMed</a></p></li>
<li><p>Ishii N, Nakahigashi K, Baba T, Robert M, Soga T, Kanai A, Hirasawa
T, Naba M, Hirai K, Hoque A, Ho PY, Kakazu Y, Sugawara K, Igarashi
S, Harada S, Masuda T, Sugiyama N, Togashi T, Hasegawa M, Takai Y,
Yugi K, Arakawa K, Iwata N, Toya Y, Nakayama Y, Nishioka T, Shimizu
K, Mori H, Tomita M. Multiple high-throughput analyses monitor the
response of E. coli to perturbations. <em>Science</em>
<strong>316</strong>(5824):593-7 (2007)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/17379776">PubMed</a></p></li>
<li><p>Kinoshita A, Tsukada K, Soga T, Hishiki T, Ueno Y, Nakayama Y,
Tomita M, Suematsu M. Roles of hemoglobin Allostery in
hypoxia-induced metabolic alterations in erythrocytes: simulation
and its verification by metabolome analysis. <em>J Biol Chem</em>
<strong>282</strong>(14):10731-41 (2007)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/17289676">PubMed</a></p></li>
<li><p>Kinoshita A, Nakayama Y, Kitayama T, Tomita M. Simulation study of
methemoglobin reduction in erythrocytes. Differential contributions
of two pathways to tolerance to oxidative stress. <em>FEBS J</em>
<strong>274</strong>(6):1449-58 (2007)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/17489100">PubMed</a></p></li>
<li><p>Itoh H, Naito Y, Tomita M. Simulation of developmental changes in
action potentials with ventricular cell models. <em>Syst Synth Biol</em>
<strong>1</strong>(1):11-23 (2007)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/19003434">PubMed</a></p></li>
<li><p>Miyoshi F, Nakayama Y, Kaizu K, Iwasaki H, Tomita M. A mathematical
model for the Kai-protein-based chemical oscillator and clock gene
expression rhythms in cyanobacteria. <em>J Biol Rhythms</em>
<strong>22</strong>(1):69-80 (2007)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/17229926">PubMed</a></p></li>
<li><p>Kitayama T, Kinoshita A, Sugimoto M, Nakayama Y, Tomita M. A
simplified method for power-law modelling of metabolic pathways from
time-course data and steady-state flux profiles. <em>Theor Biol Med
Model</em> <strong>3</strong>:24 (2006)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/16846504">PubMed</a></p></li>
<li><p>Arakawa K, Yamada Y, Shinoda K, Nakayama Y, Tomita M. GEM System:
automatic prototyping of cell-wide metabolic pathway models
from genomes. <em>BMC Bioinformatics</em> <strong>7</strong>:168 (2006)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/16553966">PubMed</a></p></li>
<li><p>Yugi K, Nakayama Y, Kojima S, Kitayama T, Tomita M. A microarray
data-based semi-kinetic method for predicting quantitative dynamics
of genetic networks. <em>BMC Bioinformatics</em> <strong>6</strong>:299 (2005)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/16351711">PubMed</a></p></li>
<li><p>Yugi K, Nakayama Y, Kinoshita A, Tomita M. Hybrid dynamic/static
method for large-scale simulation of metabolism. <em>Theor Biol Med
Model</em> <strong>2</strong>:42 (2005)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/16202166">PubMed</a></p></li>
<li><p>Nakayama Y, Kinoshita A, Tomita M. Dynamic simulation of red blood
cell metabolism and its application to the analysis of a
pathological condition. <em>Theor Biol Med Model</em> <strong>2</strong>:18 (2005)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/15882454">PubMed</a></p></li>
<li><p>Takahashi K, Arjunan SN, Tomita M. Space in systems biology of
signaling pathways–towards intracellular molecular crowding
in silico. <em>FEBS Lett</em> <strong>579</strong>(8):1783-8 (2005)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/15763552">PubMed</a></p></li>
<li><p>Ishii N, Robert M, Nakayama Y, Kanai A, Tomita M. Toward large-scale
modeling of the microbial cell for computer simulation. <em>J
Biotechnol</em> <strong>113</strong>(1-3):281-94 (2004)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/15380661">PubMed</a></p></li>
<li><p>Yugi K, Tomita M. A general computational model of mitochondrial
metabolism in a whole organelle scale. <em>Bioinformatics</em>
<strong>20</strong>(4):1795-6 (2004)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/14962921">PubMed</a>
<a class="reference external" href="http://www.e-cell.org/wp-content/uploads/2012/05/yugi-tomita-supplement.pdf">Supplement</a></p></li>
<li><p>Takahashi K, Kaizu K, Hu B, Tomita M. A multi-algorithm,
multi-timescale method for cell simulation. <em>Bioinformatics</em>
<strong>20</strong>(4):538-46 (2004)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/14990450">PubMed</a>\</p>
<p><a class="reference external" href="http://www.e-cell.org/wp-content/uploads/2012/05/takahashi03-1-supp.pdf">Supplement</a></p>
</li>
<li><p>Takahashi K, Ishikawa N, Sadamoto Y, Sasamoto H, Ohta S, Shiozawa A,
Miyoshi F, Naito Y, Nakayama Y, Tomita M. E-Cell 2: multi-platform
E-Cell simulation system. <em>Bioinformatics</em> <strong>19</strong>(13):1727-9 (2003)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/15593410">PubMed</a></p></li>
<li><p>Hucka M, Finney A, Sauro HM, Bolouri H, Doyle JC, Kitano H, Arkin
AP, Bornstein BJ, Bray D, Cornish-Bowden A, Cuellar AA, Dronov S,
Gilles ED, Ginkel M, Gor V, Goryanin II, Hedley WJ, Hodgman TC,
Hofmeyr JH, Hunter PJ, Juty NS, Kasberger JL, Kremling A, Kummer U,
Le Novère N, Loew LM, Lucio D, Mendes P, Minch E, Mjolsness ED,
Nakayama Y, Nelson MR, Nielsen PF, Sakurada T, Schaff JC, Shapiro
BE, Shimizu TS, Spence HD, Stelling J, Takahashi K, Tomita M, Wagner
J, Wang J; SBML Forum. The systems biology markup language (SBML): a
medium for representation and exchange of biochemical
network models. <em>Bioinformatics</em> <strong>19</strong>(4):524-31 (2003)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/12611808">PubMed</a></p></li>
<li><p>Tomita M, Hashimoto K, Takahashi K, Shimizu TS, Matsuzaki Y, Miyoshi
F, Saito K, Tanida S, Yugi K, Venter JC, Hutchison CA 3rd. E-CELL:
software environment for whole-cell simulation. <em>Bioinformatics</em>
<strong>15</strong>(1):72-84 (1999)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/10068694">PubMed</a></p></li>
<li><p>Tomita M, Hashimoto K, Takahashi K, Shimizu T, Matsuzaki Y, Miyoshi
F, Saito K, Tanida S, Yugi K, Venter JC, Hutchison CA. E-CELL:
Software Environment for Whole Cell Simulation. <em>Genome Inform Ser
Workshop Genome Inform</em> <strong>8</strong>:147-155 (1997)
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/11072314">PubMed</a></p></li>
</ul>
</div>


          </div>
          <div class="page-nav">
            <div class="inner"><ul class="page-nav">
  <li class="prev">
    <a href="projects/spatiocyte.html"
       title="previous chapter">← Spatiocyte</a>
  </li>
  <li class="next">
    <a href="community/members.html"
       title="next chapter">Members →</a>
  </li>
</ul><div class="footer" role="contentinfo">
      &#169; Copyright 2020, Kazunari Kaizu.
    <br>
    Created using <a href="http://sphinx-doc.org/">Sphinx</a> 3.0.3 with <a href="https://github.com/schettino72/sphinx_press_theme">Press Theme</a>.
</div>
            </div>
          </div>
      </page>
    </div>
    
    
  </body>
</html>